TestFoundation/TestNSProcessInfo.swift

```diff
@@ -0,0 +1,35 @@
+// This source file is part of the Swift.org open source project
+//
+// Copyright (c) 2014 - 2015 Apple Inc. and the Swift project authors
+// Licensed under Apache License v2.0 with Runtime Library Exception
+//
+// See http://swift.org/LICENSE.txt for license information
+// See http://swift.org/CONTRIBUTORS.txt for the list of Swift project authors
+//
+
+
+#if DEPLOYMENT_RUNTIME_OBJC || os(Linux)
+import Foundation
+import XCTest
+#else
+import SwiftFoundation
+import SwiftXCTest
+#endif
+
+
+class TestNSProcessInfo : XCTestCase {
+    
+    var allTests : [(String, () -> ())] {
+        return [
+            ("test_operatingSystemVersion", test_operatingSystemVersion ),
+        ]
+    }
+    
+    func test_operatingSystemVersion() {
+        let processInfo = NSProcessInfo.processInfo()
+        let versionString = processInfo.operatingSystemVersionString
+        let version = processInfo.operatingSystemVersion
+        let reconstructedVersionString = "Version \(version.majorVersion).\(version.minorVersion).\(version.patchVersion)"
+        XCTAssert(versionString.hasPrefix(reconstructedVersionString))
```

Comment:
I don't follow. `NSOperatingSystemVersion` has 3 *integers*, not strings. Although I agree that this test risks being incorrect, say for `10.11` where the patch is omitted from the version string.

I guess we could test that `major + minor + patch > 0`?